src/components/v2/V2Create/FloatingSaveButton.tsx

```diff
@@ -37,16 +38,20 @@ export default function FloatingSaveButton({
           {/* If no text given, can assume it’s the last tab 
           in which case we want to show the review and deploy button */}
           {text ? (
-            <Button
-              htmlType="submit"
-              type="primary"
-              onClick={() => {
-                if (onClick) onClick()
-                window.scrollTo(0, 0)
-              }}
-            >
-              <Trans>{text}</Trans>
-            </Button>
+            <div>
```

Comment:
There's some changes I want to make to this file. Making this comment to remind me to do it tomorrow before merging 🙏